test/functional/http/post/1.second.secret.js

```diff
@@ -0,0 +1,438 @@
+
+				return sendTransactionPromise(transaction).then(function (res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.equal('Sender does not have a second signature');
+					badTransactions.push(transaction);
+				});
+			});
+		});
+
+		describe('type 3 - voting delegate', function () {
+
+			it('using correct second passphrase should fail', function () {
+				transaction = node.lisk.vote.createVote(account.password, ['+' + node.eAccount.publicKey], account.secondPassword);
+
+				return sendTransactionPromise(transaction).then(function (res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.equal('Sender does not have a second signature');
+					badTransactions.push(transaction);
+				});
+			});
+		});
+
+		describe('type 4 - registering multisignature account', function () {
+
+			it('using correct second passphrase should fail', function () {
+				transaction = node.lisk.multisignature.createMultisignature(account.password, account.secondPassword, ['+' + node.eAccount.publicKey, '+' + accountNoFunds.publicKey], 1, 2);
+
+				return sendTransactionPromise(transaction).then(function (res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.equal('Sender does not have a second signature');
+					badTransactions.push(transaction);
+				});
+			});
+		});
+
+		describe('type 5 - registering dapp', function () {
+
+			it('using correct empty second passphrase should fail', function () {
+				transaction = node.lisk.dapp.createDapp(accountEmptySecondPassword.password, accountEmptySecondPassword.secondPassword, node.randomApplication());
+
+				return sendTransactionPromise(transaction).then(function (res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.equal('Sender does not have a second signature');
+					badTransactions.push(transaction);
+				});
+			});
+		});
+	});	
+
+	describe('confirmation', function () {
+
+		shared.confirmationPhase(goodTransactions, badTransactions);
+	});
+
+	describe('validation', function () {
+
+		it('using second passphrase on a fresh account should fail', function () {
+			transaction = node.lisk.transaction.createTransaction(node.eAccount.address, 1, accountNoSecondPassword.password, accountNoSecondPassword.secondPassword);
+
+			return sendTransactionPromise(transaction).then(function (res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.equal('Sender does not have a second signature');
+				badTransactionsEnforcement.push(transaction);
+			});
+		});
+
+		describe('type 0 - sending funds', function () {
+
+			it('using no second passphrase should fail', function () {
```

Comment:
using no second passphrase on account with second passphrase enabled should fail